subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ArtifactDependenciesIntegrationTest.groovy

```diff
@@ -350,10 +348,16 @@ dependencies {
     compile "org.gradle.test:lib:1.0@zip"
     compile "org.gradle.test:dist:1.0"
 }
+
+// Resolve outside of TaskAction to work with Configuration Cache
+assert configurations.compile.files.collect { it.name } == ['lib-1.0.jar', 'lib-1.0-classifier.jar', 'lib-1.0.zip', 'dist-1.0.zip']
+ArtifactView lenientView = configurations.compile.getIncoming().artifactView(view -> {
+    view.setLenient(true)
```

Comment:
Why do we need the lenient view here if the original test did not use a lenientConfiguration?